src/bionics_ui.cpp

```diff
@@ -635,7 +635,7 @@ void player::power_bionics()
                     g->refresh_all();
                     redraw = true;
                 } else {
-                    popup( _( "You can't toggle safe fuel mod on a non fueled CBM" ) );
+                    popup( _( "You can't toggle fuel saving mod on a non fueled CBM" ) );
```

Comment:
Thanks for taking my suggestion into consideration. It still says "mod" instead of "mode", however. Is it intentional? "Mod" generally means "modification" - i.e. some added parts or mechanisms. If it's just a toggleable setting "mode" would be more appropriate, similar to "power saving mode" on laptops and smartphones.